frontend/src/components/@common/MessageTextInput/MessageTextInput.tsx

```diff
@@ -1,22 +1,33 @@
 import classNames from "classnames";
-import PropTypes from "prop-types";
 import Description from "../Description/Description";
 import Label from "../Label/Label";
 import TextInput from "../TextInput/TextInput";
 import styles from "./MessageTextInput.module.css";
 
+export type MessageTextInputProps = Omit<React.InputHTMLAttributes<HTMLInputElement>, "type"> & {
+  className?: string;
+  label?: string;
+  required?: boolean;
+  description?: string;
+  value?: string;
+  name: string;
+  maxLength?: number;
+  errorMessage?: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
c: 
요렇게도 가능합니다. 저는 아래와 같이 작성하는 게 편하던데 어떠신가요?

```suggestion
  onChange: React.ChangeEventHandler<HTMLInputElement>;
```

Reply:
🤔...저는 함수의 매개변수와 반환 타입을 명시적으로 표시하는 것이 좋다고 생각해서 기존 방식을 유지하긴 했습니다. 
다른 분들의 의견도 들어보고 맞춰보도록 하겠습니다.

Reply:
저는 특별한 이유는 없고 개인적으로 리액트에서 제공하는 타입들을 쓰려고하는 편이라 `React.ChangeEventHandler`를 선호합니다.

Reply:
반영했습니다. fa46bd7